config/setupcfg.py

```diff
@@ -27,28 +28,46 @@
                 default=2379,
                 help='Host Port Number to use for etcd communication'),
     cfg.StrOpt('host_etcd_ca_cert',
-                default=None,
-                help='CA certificate location'),
+               default=None,
+               help='CA certificate location'),
     cfg.StrOpt('host_etcd_client_cert',
-                default=None,
-                help='Client certificate location'),
+               default=None,
+               help='Client certificate location'),
     cfg.StrOpt('host_etcd_client_key',
-                default=None,
-                help='Client certificate key location'),
+               default=None,
+               help='Client certificate key location'),
     cfg.StrOpt('logging',
                default='WARNING',
                help='Debug level for hpe docker volume plugin'),
     cfg.BoolOpt('use_multipath',
-               default=False,
-               help='Toggle use of multipath for volume attachments.'),
+                default=False,
+                help='Toggle use of multipath for volume attachments.'),
     cfg.BoolOpt('enforce_multipath',
-               default=False,
-               help='Toggle enforcing of multipath for volume attachments.'),
+                default=False,
+                help='Toggle enforcing of multipath for volume attachments.'),
+    cfg.BoolOpt('strict_ssh_host_key_policy',
+                default=False,
+                help='Option to enable strict host key checking.  When '
+                     'set to "True" the plugin will only connect to systems '
+                     'with a host key present in the configured '
+                     '"ssh_hosts_key_file".  When set to "False" the host key '
+                     'will be saved upon first connection and used for '
+                     'subsequent connections.  Default=False'),
+    cfg.StrOpt('state_path',
+               default='/var/lib/docker',
```

Comment:
This may not be the right default value. Need to check if environment variable can be specified in which case $HOME/.ssh would be more appropriate value.

Reply:
you can default the `state_path` value to `/root./ssh/`

Reply:
Done!